tools/Dashboard/DevHome.Dashboard/Views/DashboardView.xaml

```diff
@@ -99,7 +101,8 @@
                             <ItemsPanelTemplate>
                                 <controls:WidgetBoard XYFocusKeyboardNavigation="Enabled"
                                                       XYFocusRightNavigationStrategy="RectilinearDistance"
-                                                      XYFocusLeftNavigationStrategy="RectilinearDistance"/>
+                                                      XYFocusLeftNavigationStrategy="RectilinearDistance"
+                                                      KeyUp="HandleKeyUpAsync" />

```

Comment:
Would promoting this to a command prevent any unexpected concurrent asynchronous execution behavior?

Reply:
Because it's in a ItemsPanelTempate, I can't bind to the command 🙁
https://github.com/microsoft/microsoft-ui-xaml/issues/2508